alibi_detect/utils/pytorch/prediction.py

```diff
@@ -90,7 +90,7 @@ def predict_batch_transformer(x: Union[np.ndarray, torch.Tensor], model: Union[n
         for i in range(n_minibatch):
             istart, istop = i * batch_size, min((i + 1) * batch_size, n)
             tokens = tokenizer.batch_encode_plus(  # type: ignore
```

Comment:
The transformer docs say `batch_encode_plus` is deprecated and `__call__` should be used: https://huggingface.co/transformers/internal/tokenization_utils.html?highlight=batch_encode_plus#transformers.tokenization_utils_base.PreTrainedTokenizerBase.batch_encode_plus